src/opening-period/description/OpeningPeriodDescription.scss

```diff
@@ -0,0 +1,23 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.add-new-opening-period-title {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+  font-size: $fontsize-body-l;
+  line-height: 2.5;
```

Comment:
Those line-heights seem to be too small:

![image](https://user-images.githubusercontent.com/2777633/101773283-44c39900-3af5-11eb-8374-6dd63a94e6a1.png)

Reply:
Or should this be replaced with HDS-text-input?

Reply:
Does it play well with react-form-hook?

Reply:
We probably could replace it with HDS-text-input element, though it's little outside of the scope of this PR, since the purpose was to continue the form, even though I did some refactoring to the form. Maybe I'll create a jira issue about it?